cityofaustin/atd-moped
# Update dependencies for Moped. Minor + patch upgrades only

## Associated issues

https://github.com/cityofaustin/atd-data-tech/issues/15204

## Process

After working through getting the versions as up-to-date based on the instructions in the issue, I ran through the release testing script, and didn't find any regressions from this update. I found two other issues, and spun those off into new issues as they were not related to this work.

### Next Steps

I briefly explored further upgrades, but I quickly ran into serious road-blocks with major, breaking changes that some of our libraries have done. One of the big ones that made me back away slowly was the the two major releases of `aws-amplify` which is integral to our auth system. When they say breaking change, they mean it. It'd be a kinda-rewrite-it-fresh on our auth components I fear. Not impossible, but not trivial by any means.

## Testing

* Local & https://deploy-preview-1243--atd-moped-main.netlify.app/moped/ both


### Steps to test

* optionally:
  * `rm -frv node_modules` if you want to start fresh, like a netlify build
    * `rm package-lock.json` too if you want to try verbose-mode. Should work fine though, but you'll see errors that we've seen in the past but `npm install` masks on subsequent runs.

* can you, in order:
  * `nvm use`
  * `npm install`
  * `npm start`

* **_everything_** in the application is in-bounds for testing this. 
  * can you break anything?

---
#### Ship list
